rest-api-spec/test/cluster.state/40_allow_no_indices.yaml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
I was wondering if it makes sense to merge this file with the previous one, since they both test indices options?